strapi/strapi
# fix(content-releases): add edit entry to details view

### What does it do?

Add a missing option on the Release Details View to go to the edit page of an specific entry. 

To be able to do this also fix a problem on `CheckPermissions` component from helper-plugin: Now it's able to also check permissions on locales.

### How to test it

First you need to have content-releases enabled with a EE license.

1. Set permissions in a role to read and edit releases.
2. Make sure to have at least 2 locales and set permissions to one user to update one locale of one content type but the other not.
3. Add two entries from these different locales to a release
4. Go to the release details page, when clicking on the 3 dots button you should see the "Edit entry" option on the one from the locale you have permissions to update
